public/js/controllers/inventoryCtrl.js

```diff
@@ -91,6 +98,16 @@ habitrpg.controller("InventoryCtrl",
         User.user.achievements.beastMaster = true;
         $rootScope.openModal('achievements/beastMaster');
       }
+
+      // Checks if Triad Bingo has been reached for the first time
+      if(!User.user.achievements.triadBingo
+          && Shared.countPets(null, User.user.items.pets) >= 90
+          && Shared.countMounts(null, User.user.items.mounts) >= 90
+          && $scope.calcTriadBingo()) {
```

Comment:
Should `$scope.calcTriadBingo()` just be a function local to $scope.hatch? 

Also, I suppose I could include the `countPets` and `countMounts` checks in it, so the if would just be `(!User.user.achievements.triadBingo && calcTriadBingo())`